plans/chew-large-datasets/main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"github.com/ipfs/testground/sdk/runtime"
+	test "github.com/ipfs/testground/plans/chew-large-datasets/test"
+)
+
+var testCases = []func(*runtime.RunEnv){
+	test.IpfsAddDefaults,
```

Comment:
❓ I'm trying to run this one manually with:

```
> TEST_CASE="IpfsAddDefaults" go run main.go
panic: test case sequence number not set
```

Assuming that I'm reading https://github.com/ipfs/testground/blob/master/sdk/runtime/runenv.go#L20 right. As you can see, it fails. Any tip please? :)

Reply:
Update: learned through -- -- that TEST_CASE_SEQ is needed for the test selection. Now getting it to work locally

```
> TEST_CASE="ipfs-add-defaults" TEST_CASE_SEQ="0" go run main.go
Yo
{"runenv":{"test_plan":"","test_case":"ipfs-add-defaults","test_run":"","test_seq":0,"test_instance_count":-1},"timestamp":1570110821332210000,"result":{"outcome":"ok"}}
```

This means that the test case is actually being picked by the order in which they show up testCases array rather than the name. (wasn't obvious)